src/util.c

```diff
@@ -131,6 +131,17 @@ static unsigned long id_or_max(const char *name)
 	return id;
 }
 
+bool hyper_is_id(const char *name, unsigned long *val)
```

Comment:
The name of this function is confusing - maybe `hyper_name_to_id()` would be clearer? Also, how about a comment header explaining what it does?